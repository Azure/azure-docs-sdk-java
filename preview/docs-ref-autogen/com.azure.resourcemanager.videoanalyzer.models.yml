### YamlMime:JavaPackage
uid: "com.azure.resourcemanager.videoanalyzer.models"
fullName: "com.azure.resourcemanager.videoanalyzer.models"
name: "com.azure.resourcemanager.videoanalyzer.models"
summary: "Package containing the data models for VideoAnalyzerManagementClient. Azure Video Analyzer provides a platform for\n you to build intelligent video applications that span the edge and the cloud."
classes:
- "com.azure.resourcemanager.videoanalyzer.models.AccessPolicyEccAlgo"
- "com.azure.resourcemanager.videoanalyzer.models.AccessPolicyEntityCollection"
- "com.azure.resourcemanager.videoanalyzer.models.AccessPolicyRole"
- "com.azure.resourcemanager.videoanalyzer.models.AccessPolicyRsaAlgo"
- "com.azure.resourcemanager.videoanalyzer.models.AccountEncryption"
- "com.azure.resourcemanager.videoanalyzer.models.AccountEncryptionKeyType"
- "com.azure.resourcemanager.videoanalyzer.models.ActionType"
- "com.azure.resourcemanager.videoanalyzer.models.AudioEncoderAac"
- "com.azure.resourcemanager.videoanalyzer.models.AudioEncoderBase"
- "com.azure.resourcemanager.videoanalyzer.models.AuthenticationBase"
- "com.azure.resourcemanager.videoanalyzer.models.CertificateSource"
- "com.azure.resourcemanager.videoanalyzer.models.CheckNameAvailabilityReason"
- "com.azure.resourcemanager.videoanalyzer.models.CheckNameAvailabilityRequest"
- "com.azure.resourcemanager.videoanalyzer.models.CredentialsBase"
- "com.azure.resourcemanager.videoanalyzer.models.EccTokenKey"
- "com.azure.resourcemanager.videoanalyzer.models.EdgeModuleEntityCollection"
- "com.azure.resourcemanager.videoanalyzer.models.EncoderCustomPreset"
- "com.azure.resourcemanager.videoanalyzer.models.EncoderPresetBase"
- "com.azure.resourcemanager.videoanalyzer.models.EncoderProcessor"
- "com.azure.resourcemanager.videoanalyzer.models.EncoderSystemPreset"
- "com.azure.resourcemanager.videoanalyzer.models.EncoderSystemPresetType"
- "com.azure.resourcemanager.videoanalyzer.models.Endpoint"
- "com.azure.resourcemanager.videoanalyzer.models.EndpointBase"
- "com.azure.resourcemanager.videoanalyzer.models.GroupLevelAccessControl"
- "com.azure.resourcemanager.videoanalyzer.models.IotHub"
- "com.azure.resourcemanager.videoanalyzer.models.JwtAuthentication"
- "com.azure.resourcemanager.videoanalyzer.models.KeyVaultProperties"
- "com.azure.resourcemanager.videoanalyzer.models.Kind"
- "com.azure.resourcemanager.videoanalyzer.models.ListProvisioningTokenInput"
- "com.azure.resourcemanager.videoanalyzer.models.LivePipelineCollection"
- "com.azure.resourcemanager.videoanalyzer.models.LivePipelineState"
- "com.azure.resourcemanager.videoanalyzer.models.LivePipelineUpdate"
- "com.azure.resourcemanager.videoanalyzer.models.LogSpecification"
- "com.azure.resourcemanager.videoanalyzer.models.MetricAggregationType"
- "com.azure.resourcemanager.videoanalyzer.models.MetricDimension"
- "com.azure.resourcemanager.videoanalyzer.models.MetricSpecification"
- "com.azure.resourcemanager.videoanalyzer.models.MetricUnit"
- "com.azure.resourcemanager.videoanalyzer.models.NetworkAccessControl"
- "com.azure.resourcemanager.videoanalyzer.models.NodeBase"
- "com.azure.resourcemanager.videoanalyzer.models.NodeInput"
- "com.azure.resourcemanager.videoanalyzer.models.Operation"
- "com.azure.resourcemanager.videoanalyzer.models.OperationDisplay"
- "com.azure.resourcemanager.videoanalyzer.models.ParameterDeclaration"
- "com.azure.resourcemanager.videoanalyzer.models.ParameterDefinition"
- "com.azure.resourcemanager.videoanalyzer.models.ParameterType"
- "com.azure.resourcemanager.videoanalyzer.models.PemCertificateList"
- "com.azure.resourcemanager.videoanalyzer.models.PipelineJobCollection"
- "com.azure.resourcemanager.videoanalyzer.models.PipelineJobError"
- "com.azure.resourcemanager.videoanalyzer.models.PipelineJobState"
- "com.azure.resourcemanager.videoanalyzer.models.PipelineJobUpdate"
- "com.azure.resourcemanager.videoanalyzer.models.PipelineTopologyCollection"
- "com.azure.resourcemanager.videoanalyzer.models.PipelineTopologyUpdate"
- "com.azure.resourcemanager.videoanalyzer.models.PrivateEndpoint"
- "com.azure.resourcemanager.videoanalyzer.models.PrivateEndpointConnectionProvisioningState"
- "com.azure.resourcemanager.videoanalyzer.models.PrivateEndpointConnectionsCreateOrUpdateHeaders"
- "com.azure.resourcemanager.videoanalyzer.models.PrivateEndpointConnectionsCreateOrUpdateResponse"
- "com.azure.resourcemanager.videoanalyzer.models.PrivateEndpointServiceConnectionStatus"
- "com.azure.resourcemanager.videoanalyzer.models.PrivateLinkServiceConnectionState"
- "com.azure.resourcemanager.videoanalyzer.models.ProcessorNodeBase"
- "com.azure.resourcemanager.videoanalyzer.models.Properties"
- "com.azure.resourcemanager.videoanalyzer.models.ProvisioningState"
- "com.azure.resourcemanager.videoanalyzer.models.PublicNetworkAccess"
- "com.azure.resourcemanager.videoanalyzer.models.ResourceIdentity"
- "com.azure.resourcemanager.videoanalyzer.models.RsaTokenKey"
- "com.azure.resourcemanager.videoanalyzer.models.RtspSource"
- "com.azure.resourcemanager.videoanalyzer.models.RtspTransport"
- "com.azure.resourcemanager.videoanalyzer.models.SecureIotDeviceRemoteTunnel"
- "com.azure.resourcemanager.videoanalyzer.models.ServiceSpecification"
- "com.azure.resourcemanager.videoanalyzer.models.SinkNodeBase"
- "com.azure.resourcemanager.videoanalyzer.models.Sku"
- "com.azure.resourcemanager.videoanalyzer.models.SkuName"
- "com.azure.resourcemanager.videoanalyzer.models.SkuTier"
- "com.azure.resourcemanager.videoanalyzer.models.SourceNodeBase"
- "com.azure.resourcemanager.videoanalyzer.models.StorageAccount"
- "com.azure.resourcemanager.videoanalyzer.models.TimeSequenceBase"
- "com.azure.resourcemanager.videoanalyzer.models.TlsEndpoint"
- "com.azure.resourcemanager.videoanalyzer.models.TlsValidationOptions"
- "com.azure.resourcemanager.videoanalyzer.models.TokenClaim"
- "com.azure.resourcemanager.videoanalyzer.models.TokenKey"
- "com.azure.resourcemanager.videoanalyzer.models.TunnelBase"
- "com.azure.resourcemanager.videoanalyzer.models.UnsecuredEndpoint"
- "com.azure.resourcemanager.videoanalyzer.models.UserAssignedManagedIdentity"
- "com.azure.resourcemanager.videoanalyzer.models.UsernamePasswordCredentials"
- "com.azure.resourcemanager.videoanalyzer.models.VideoAnalyzerEndpointType"
- "com.azure.resourcemanager.videoanalyzer.models.VideoAnalyzerIdentity"
- "com.azure.resourcemanager.videoanalyzer.models.VideoAnalyzerUpdate"
- "com.azure.resourcemanager.videoanalyzer.models.VideoAnalyzersCreateOrUpdateHeaders"
- "com.azure.resourcemanager.videoanalyzer.models.VideoAnalyzersCreateOrUpdateResponse"
- "com.azure.resourcemanager.videoanalyzer.models.VideoAnalyzersUpdateHeaders"
- "com.azure.resourcemanager.videoanalyzer.models.VideoAnalyzersUpdateResponse"
- "com.azure.resourcemanager.videoanalyzer.models.VideoArchival"
- "com.azure.resourcemanager.videoanalyzer.models.VideoContentUrls"
- "com.azure.resourcemanager.videoanalyzer.models.VideoCreationProperties"
- "com.azure.resourcemanager.videoanalyzer.models.VideoEncoderBase"
- "com.azure.resourcemanager.videoanalyzer.models.VideoEncoderH264"
- "com.azure.resourcemanager.videoanalyzer.models.VideoEntityCollection"
- "com.azure.resourcemanager.videoanalyzer.models.VideoFlags"
- "com.azure.resourcemanager.videoanalyzer.models.VideoMediaInfo"
- "com.azure.resourcemanager.videoanalyzer.models.VideoPreviewImageUrls"
- "com.azure.resourcemanager.videoanalyzer.models.VideoPublishingOptions"
- "com.azure.resourcemanager.videoanalyzer.models.VideoScale"
- "com.azure.resourcemanager.videoanalyzer.models.VideoScaleMode"
- "com.azure.resourcemanager.videoanalyzer.models.VideoSequenceAbsoluteTimeMarkers"
- "com.azure.resourcemanager.videoanalyzer.models.VideoSink"
- "com.azure.resourcemanager.videoanalyzer.models.VideoSource"
- "com.azure.resourcemanager.videoanalyzer.models.VideoType"
interfaces:
- "com.azure.resourcemanager.videoanalyzer.models.AccessPolicies"
- "com.azure.resourcemanager.videoanalyzer.models.AccessPolicyEntity"
- "com.azure.resourcemanager.videoanalyzer.models.AccessPolicyEntity.Definition"
- "com.azure.resourcemanager.videoanalyzer.models.AccessPolicyEntity.DefinitionStages"
- "com.azure.resourcemanager.videoanalyzer.models.AccessPolicyEntity.DefinitionStages.Blank"
- "com.azure.resourcemanager.videoanalyzer.models.AccessPolicyEntity.DefinitionStages.WithAuthentication"
- "com.azure.resourcemanager.videoanalyzer.models.AccessPolicyEntity.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.videoanalyzer.models.AccessPolicyEntity.DefinitionStages.WithParentResource"
- "com.azure.resourcemanager.videoanalyzer.models.AccessPolicyEntity.DefinitionStages.WithRole"
- "com.azure.resourcemanager.videoanalyzer.models.AccessPolicyEntity.Update"
- "com.azure.resourcemanager.videoanalyzer.models.AccessPolicyEntity.UpdateStages"
- "com.azure.resourcemanager.videoanalyzer.models.AccessPolicyEntity.UpdateStages.WithAuthentication"
- "com.azure.resourcemanager.videoanalyzer.models.AccessPolicyEntity.UpdateStages.WithRole"
- "com.azure.resourcemanager.videoanalyzer.models.CheckNameAvailabilityResponse"
- "com.azure.resourcemanager.videoanalyzer.models.EdgeModuleEntity"
- "com.azure.resourcemanager.videoanalyzer.models.EdgeModuleEntity.Definition"
- "com.azure.resourcemanager.videoanalyzer.models.EdgeModuleEntity.DefinitionStages"
- "com.azure.resourcemanager.videoanalyzer.models.EdgeModuleEntity.DefinitionStages.Blank"
- "com.azure.resourcemanager.videoanalyzer.models.EdgeModuleEntity.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.videoanalyzer.models.EdgeModuleEntity.DefinitionStages.WithParentResource"
- "com.azure.resourcemanager.videoanalyzer.models.EdgeModuleEntity.Update"
- "com.azure.resourcemanager.videoanalyzer.models.EdgeModuleEntity.UpdateStages"
- "com.azure.resourcemanager.videoanalyzer.models.EdgeModuleProvisioningToken"
- "com.azure.resourcemanager.videoanalyzer.models.EdgeModules"
- "com.azure.resourcemanager.videoanalyzer.models.LivePipeline"
- "com.azure.resourcemanager.videoanalyzer.models.LivePipeline.Definition"
- "com.azure.resourcemanager.videoanalyzer.models.LivePipeline.DefinitionStages"
- "com.azure.resourcemanager.videoanalyzer.models.LivePipeline.DefinitionStages.Blank"
- "com.azure.resourcemanager.videoanalyzer.models.LivePipeline.DefinitionStages.WithBitrateKbps"
- "com.azure.resourcemanager.videoanalyzer.models.LivePipeline.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.videoanalyzer.models.LivePipeline.DefinitionStages.WithDescription"
- "com.azure.resourcemanager.videoanalyzer.models.LivePipeline.DefinitionStages.WithParameters"
- "com.azure.resourcemanager.videoanalyzer.models.LivePipeline.DefinitionStages.WithParentResource"
- "com.azure.resourcemanager.videoanalyzer.models.LivePipeline.DefinitionStages.WithTopologyName"
- "com.azure.resourcemanager.videoanalyzer.models.LivePipeline.Update"
- "com.azure.resourcemanager.videoanalyzer.models.LivePipeline.UpdateStages"
- "com.azure.resourcemanager.videoanalyzer.models.LivePipeline.UpdateStages.WithBitrateKbps"
- "com.azure.resourcemanager.videoanalyzer.models.LivePipeline.UpdateStages.WithDescription"
- "com.azure.resourcemanager.videoanalyzer.models.LivePipeline.UpdateStages.WithParameters"
- "com.azure.resourcemanager.videoanalyzer.models.LivePipeline.UpdateStages.WithTopologyName"
- "com.azure.resourcemanager.videoanalyzer.models.LivePipelineOperationStatus"
- "com.azure.resourcemanager.videoanalyzer.models.LivePipelineOperationStatuses"
- "com.azure.resourcemanager.videoanalyzer.models.LivePipelines"
- "com.azure.resourcemanager.videoanalyzer.models.Locations"
- "com.azure.resourcemanager.videoanalyzer.models.OperationCollection"
- "com.azure.resourcemanager.videoanalyzer.models.OperationResults"
- "com.azure.resourcemanager.videoanalyzer.models.OperationStatuses"
- "com.azure.resourcemanager.videoanalyzer.models.Operations"
- "com.azure.resourcemanager.videoanalyzer.models.PipelineJob"
- "com.azure.resourcemanager.videoanalyzer.models.PipelineJob.Definition"
- "com.azure.resourcemanager.videoanalyzer.models.PipelineJob.DefinitionStages"
- "com.azure.resourcemanager.videoanalyzer.models.PipelineJob.DefinitionStages.Blank"
- "com.azure.resourcemanager.videoanalyzer.models.PipelineJob.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.videoanalyzer.models.PipelineJob.DefinitionStages.WithDescription"
- "com.azure.resourcemanager.videoanalyzer.models.PipelineJob.DefinitionStages.WithParameters"
- "com.azure.resourcemanager.videoanalyzer.models.PipelineJob.DefinitionStages.WithParentResource"
- "com.azure.resourcemanager.videoanalyzer.models.PipelineJob.DefinitionStages.WithTopologyName"
- "com.azure.resourcemanager.videoanalyzer.models.PipelineJob.Update"
- "com.azure.resourcemanager.videoanalyzer.models.PipelineJob.UpdateStages"
- "com.azure.resourcemanager.videoanalyzer.models.PipelineJob.UpdateStages.WithDescription"
- "com.azure.resourcemanager.videoanalyzer.models.PipelineJob.UpdateStages.WithParameters"
- "com.azure.resourcemanager.videoanalyzer.models.PipelineJob.UpdateStages.WithTopologyName"
- "com.azure.resourcemanager.videoanalyzer.models.PipelineJobOperationStatus"
- "com.azure.resourcemanager.videoanalyzer.models.PipelineJobOperationStatuses"
- "com.azure.resourcemanager.videoanalyzer.models.PipelineJobs"
- "com.azure.resourcemanager.videoanalyzer.models.PipelineTopologies"
- "com.azure.resourcemanager.videoanalyzer.models.PipelineTopology"
- "com.azure.resourcemanager.videoanalyzer.models.PipelineTopology.Definition"
- "com.azure.resourcemanager.videoanalyzer.models.PipelineTopology.DefinitionStages"
- "com.azure.resourcemanager.videoanalyzer.models.PipelineTopology.DefinitionStages.Blank"
- "com.azure.resourcemanager.videoanalyzer.models.PipelineTopology.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.videoanalyzer.models.PipelineTopology.DefinitionStages.WithDescription"
- "com.azure.resourcemanager.videoanalyzer.models.PipelineTopology.DefinitionStages.WithKind"
- "com.azure.resourcemanager.videoanalyzer.models.PipelineTopology.DefinitionStages.WithParameters"
- "com.azure.resourcemanager.videoanalyzer.models.PipelineTopology.DefinitionStages.WithParentResource"
- "com.azure.resourcemanager.videoanalyzer.models.PipelineTopology.DefinitionStages.WithProcessors"
- "com.azure.resourcemanager.videoanalyzer.models.PipelineTopology.DefinitionStages.WithSinks"
- "com.azure.resourcemanager.videoanalyzer.models.PipelineTopology.DefinitionStages.WithSku"
- "com.azure.resourcemanager.videoanalyzer.models.PipelineTopology.DefinitionStages.WithSources"
- "com.azure.resourcemanager.videoanalyzer.models.PipelineTopology.Update"
- "com.azure.resourcemanager.videoanalyzer.models.PipelineTopology.UpdateStages"
- "com.azure.resourcemanager.videoanalyzer.models.PipelineTopology.UpdateStages.WithDescription"
- "com.azure.resourcemanager.videoanalyzer.models.PipelineTopology.UpdateStages.WithKind"
- "com.azure.resourcemanager.videoanalyzer.models.PipelineTopology.UpdateStages.WithParameters"
- "com.azure.resourcemanager.videoanalyzer.models.PipelineTopology.UpdateStages.WithProcessors"
- "com.azure.resourcemanager.videoanalyzer.models.PipelineTopology.UpdateStages.WithSinks"
- "com.azure.resourcemanager.videoanalyzer.models.PipelineTopology.UpdateStages.WithSku"
- "com.azure.resourcemanager.videoanalyzer.models.PipelineTopology.UpdateStages.WithSources"
- "com.azure.resourcemanager.videoanalyzer.models.PrivateEndpointConnection"
- "com.azure.resourcemanager.videoanalyzer.models.PrivateEndpointConnection.Definition"
- "com.azure.resourcemanager.videoanalyzer.models.PrivateEndpointConnection.DefinitionStages"
- "com.azure.resourcemanager.videoanalyzer.models.PrivateEndpointConnection.DefinitionStages.Blank"
- "com.azure.resourcemanager.videoanalyzer.models.PrivateEndpointConnection.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.videoanalyzer.models.PrivateEndpointConnection.DefinitionStages.WithParentResource"
- "com.azure.resourcemanager.videoanalyzer.models.PrivateEndpointConnection.DefinitionStages.WithPrivateEndpoint"
- "com.azure.resourcemanager.videoanalyzer.models.PrivateEndpointConnection.DefinitionStages.WithPrivateLinkServiceConnectionState"
- "com.azure.resourcemanager.videoanalyzer.models.PrivateEndpointConnection.Update"
- "com.azure.resourcemanager.videoanalyzer.models.PrivateEndpointConnection.UpdateStages"
- "com.azure.resourcemanager.videoanalyzer.models.PrivateEndpointConnection.UpdateStages.WithPrivateEndpoint"
- "com.azure.resourcemanager.videoanalyzer.models.PrivateEndpointConnection.UpdateStages.WithPrivateLinkServiceConnectionState"
- "com.azure.resourcemanager.videoanalyzer.models.PrivateEndpointConnectionListResult"
- "com.azure.resourcemanager.videoanalyzer.models.PrivateEndpointConnections"
- "com.azure.resourcemanager.videoanalyzer.models.PrivateLinkResource"
- "com.azure.resourcemanager.videoanalyzer.models.PrivateLinkResourceListResult"
- "com.azure.resourcemanager.videoanalyzer.models.PrivateLinkResources"
- "com.azure.resourcemanager.videoanalyzer.models.VideoAnalyzer"
- "com.azure.resourcemanager.videoanalyzer.models.VideoAnalyzer.Definition"
- "com.azure.resourcemanager.videoanalyzer.models.VideoAnalyzer.DefinitionStages"
- "com.azure.resourcemanager.videoanalyzer.models.VideoAnalyzer.DefinitionStages.Blank"
- "com.azure.resourcemanager.videoanalyzer.models.VideoAnalyzer.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.videoanalyzer.models.VideoAnalyzer.DefinitionStages.WithEncryption"
- "com.azure.resourcemanager.videoanalyzer.models.VideoAnalyzer.DefinitionStages.WithIdentity"
- "com.azure.resourcemanager.videoanalyzer.models.VideoAnalyzer.DefinitionStages.WithIotHubs"
- "com.azure.resourcemanager.videoanalyzer.models.VideoAnalyzer.DefinitionStages.WithLocation"
- "com.azure.resourcemanager.videoanalyzer.models.VideoAnalyzer.DefinitionStages.WithNetworkAccessControl"
- "com.azure.resourcemanager.videoanalyzer.models.VideoAnalyzer.DefinitionStages.WithPublicNetworkAccess"
- "com.azure.resourcemanager.videoanalyzer.models.VideoAnalyzer.DefinitionStages.WithResourceGroup"
- "com.azure.resourcemanager.videoanalyzer.models.VideoAnalyzer.DefinitionStages.WithStorageAccounts"
- "com.azure.resourcemanager.videoanalyzer.models.VideoAnalyzer.DefinitionStages.WithTags"
- "com.azure.resourcemanager.videoanalyzer.models.VideoAnalyzer.Update"
- "com.azure.resourcemanager.videoanalyzer.models.VideoAnalyzer.UpdateStages"
- "com.azure.resourcemanager.videoanalyzer.models.VideoAnalyzer.UpdateStages.WithEncryption"
- "com.azure.resourcemanager.videoanalyzer.models.VideoAnalyzer.UpdateStages.WithIdentity"
- "com.azure.resourcemanager.videoanalyzer.models.VideoAnalyzer.UpdateStages.WithIotHubs"
- "com.azure.resourcemanager.videoanalyzer.models.VideoAnalyzer.UpdateStages.WithNetworkAccessControl"
- "com.azure.resourcemanager.videoanalyzer.models.VideoAnalyzer.UpdateStages.WithPublicNetworkAccess"
- "com.azure.resourcemanager.videoanalyzer.models.VideoAnalyzer.UpdateStages.WithStorageAccounts"
- "com.azure.resourcemanager.videoanalyzer.models.VideoAnalyzer.UpdateStages.WithTags"
- "com.azure.resourcemanager.videoanalyzer.models.VideoAnalyzerCollection"
- "com.azure.resourcemanager.videoanalyzer.models.VideoAnalyzerOperationResults"
- "com.azure.resourcemanager.videoanalyzer.models.VideoAnalyzerOperationStatus"
- "com.azure.resourcemanager.videoanalyzer.models.VideoAnalyzerOperationStatuses"
- "com.azure.resourcemanager.videoanalyzer.models.VideoAnalyzerPrivateEndpointConnectionOperationStatus"
- "com.azure.resourcemanager.videoanalyzer.models.VideoAnalyzers"
- "com.azure.resourcemanager.videoanalyzer.models.VideoContentToken"
- "com.azure.resourcemanager.videoanalyzer.models.VideoEntity"
- "com.azure.resourcemanager.videoanalyzer.models.VideoEntity.Definition"
- "com.azure.resourcemanager.videoanalyzer.models.VideoEntity.DefinitionStages"
- "com.azure.resourcemanager.videoanalyzer.models.VideoEntity.DefinitionStages.Blank"
- "com.azure.resourcemanager.videoanalyzer.models.VideoEntity.DefinitionStages.WithArchival"
- "com.azure.resourcemanager.videoanalyzer.models.VideoEntity.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.videoanalyzer.models.VideoEntity.DefinitionStages.WithDescription"
- "com.azure.resourcemanager.videoanalyzer.models.VideoEntity.DefinitionStages.WithParentResource"
- "com.azure.resourcemanager.videoanalyzer.models.VideoEntity.DefinitionStages.WithTitle"
- "com.azure.resourcemanager.videoanalyzer.models.VideoEntity.Update"
- "com.azure.resourcemanager.videoanalyzer.models.VideoEntity.UpdateStages"
- "com.azure.resourcemanager.videoanalyzer.models.VideoEntity.UpdateStages.WithArchival"
- "com.azure.resourcemanager.videoanalyzer.models.VideoEntity.UpdateStages.WithDescription"
- "com.azure.resourcemanager.videoanalyzer.models.VideoEntity.UpdateStages.WithTitle"
- "com.azure.resourcemanager.videoanalyzer.models.Videos"
metadata: {}
package: "com.azure.resourcemanager.videoanalyzer.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-videoanalyzer:1.0.0-beta.3
