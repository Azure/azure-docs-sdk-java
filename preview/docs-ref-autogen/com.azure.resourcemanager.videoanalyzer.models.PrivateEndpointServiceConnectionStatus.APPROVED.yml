### YamlMime:JavaMember
uid: "com.azure.resourcemanager.videoanalyzer.models.PrivateEndpointServiceConnectionStatus.APPROVED*"
fullName: "com.azure.resourcemanager.videoanalyzer.models.PrivateEndpointServiceConnectionStatus.APPROVED"
name: "APPROVED"
nameWithType: "PrivateEndpointServiceConnectionStatus.APPROVED"
members:
- uid: "com.azure.resourcemanager.videoanalyzer.models.PrivateEndpointServiceConnectionStatus.APPROVED"
  fullName: "com.azure.resourcemanager.videoanalyzer.models.PrivateEndpointServiceConnectionStatus.APPROVED"
  name: "APPROVED"
  nameWithType: "PrivateEndpointServiceConnectionStatus.APPROVED"
  summary: "Static value Approved for PrivateEndpointServiceConnectionStatus."
  syntax: "public static final PrivateEndpointServiceConnectionStatus APPROVED"
type: "field"
metadata: {}
package: "com.azure.resourcemanager.videoanalyzer.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-videoanalyzer:1.0.0-beta.3
